core/hooks/allocation_source.py

```diff
@@ -28,33 +28,13 @@ def listen_for_allocation_overage(sender, instance, raw, **kwargs):
         return None
     # Circular dep...
     from core.models import EventTable
-    from service.tasks.monitoring import enforce_allocation_overage
     payload = event.payload
     allocation_source_id = payload['allocation_source_id']
     new_compute_used = payload['compute_used']
     source = AllocationSource.objects.filter(source_id=allocation_source_id).first()
     prev_enforcement_event = EventTable.objects\
         .filter(name="allocation_source_threshold_enforced")\
         .filter(entity_id=allocation_source_id).last()
-    # test for previous event of 'allocation_source_threshold_enforced'
```

Comment:
All of this is pointless if we are not enforcing the quotas (enforce_allocation_overage).